bindings/java/src/main/java/org/apache/opendal/BlockingOperator.java

```diff
@@ -92,6 +95,14 @@ public void rename(String sourcePath, String targetPath) {
         rename(nativeHandle, sourcePath, targetPath);
     }
 
+    public void removeAll(String path) {
+        removeAll(nativeHandle, path);
+    }
+
+    public List<Entry> list(String path) {
+        return new ArrayList<>(Arrays.asList(list(nativeHandle, path)));
```

Comment:
```suggestion
        return Arrays.asList(list(nativeHandle, path));
```

Redundant?

Reply:
Should the user be allowed to `add` and `remove` operations to the returned list? 🤔

Reply:
I can't find a scenario where users would need to change the list we provide.

Reply:
> I can't find a scenario where users would need to change the list we provide.

And users can construct their own mutable list on demand. Just as we did it internally in this patch :D